pkg/apis/core/types.go

```diff
@@ -2953,6 +2953,25 @@ func (ps *PodSpec) Workloads() []CommonInfo {
 	return ps.WorkloadInfo
 }
 
+func (ps *PodSpec) SetWorkloads() {
+	// Update mutable fields
```

Comment:
nit: removable comment.

Reply:
this is not on apiserver, rather on local object. that's why the comment to clarify.